src/main/scala/org/constellation/primitives/Transaction.scala

```diff
@@ -62,7 +62,7 @@ case class Transaction(
 
   def hash: String = edge.observationEdge.hash
 
-  def getPrevTxHash = edge.signedObservationEdge.getHexEncoding
+  def getNextPrevTxHash = edge.signedObservationEdge.getHexEncoding
```

Comment:
so get `next` or `prev`? it's kinda confusing

Reply:
Yeah, I like getNextPrevTxHash because it's the PrevTxHash for the 'next' transaction. But I think we should change lastTxRef.prevHash to lastTxRef.prevHashRef. Wyt?

Reply:
works for me